docs/plugins/irods_rule_language.md

```diff
@@ -9,9 +9,11 @@ to allow complex policies or actions to be defined.
 Everything is a rule in the iRODS Rule Language. A typical rule looks like:
 
 ~~~
-acPostProcForPut {
-  on($objPath like "*.txt") {
-    msiDataObjCopy($objPath,"$objPath.copy");
+pep_api_data_obj_put_post(*INSTANCE_NAME, *COMM, *DATAOBJINP, *BUFFER, *PORTAL_OPR_OUT) {
+  *logical_path = *DATAOBJINP.obj_path;
+
+  if (*logical_path like "*.txt") {
+    msiDataObjCopy(*logical_path, "*logical_path.copy", "", *ec);
```

Comment:
need to update the paragraph below this to be true

Reply:
Updated. Please take a look.